specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,3027 @@
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeTextTaskResultsKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeTextLROResultsKind": {
+      "type": "string",
+      "description": "Enumeration of supported Text Analysis long-running operation task results.",
+      "enum": [
+        "SentimentAnalysisLROResults",
+        "EntityRecognitionLROResults",
+        "PiiEntityRecognitionLROResults",
+        "KeyPhraseExtractionLROResults",
+        "LanguageDetectionLROResults",
+        "EntityLinkingLROResults",
+        "HealthcareLROResults",
+        "ExtractiveSummarizationLROResults",
+        "CustomEntityRecognitionLROResults",
+        "CustomSingleClassificationLROResults",
+        "CustomMultiClassificationLROResults"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeTextLROResultsKind",
+        "modelAsString": true
+      }
+    },
+    "MultiLanguageAnalysisInput": {
+      "properties": {
+        "documents": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/MultiLanguageInput"
+          }
+        }
+      }
+    },
+    "LanguageDetectionAnalysisInput": {
+      "properties": {
+        "documents": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/LanguageInput"
+          }
+        }
+      }
+    },
+    "AnalyzeTextTask": {
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeTextTaskKind"
+        }
+      }
+    },
+    "AnalyzeTextLROTask": {
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeTextLROTaskKind"
+        }
+      }
+    },
+    "AnalyzeTextTaskResult": {
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeTextTaskResultsKind"
+        }
+      }
+    },
+    "AnalyzeTextEntityLinkingInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/EntityLinkingTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "EntityLinking"
+    },
+    "AnalyzeTextEntityRecognitionInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/EntitiesTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "EntityRecognition"
+    },
+    "AnalyzeTextKeyPhraseExtractionInput": {
```

Comment:
In TA v3.2-preview.2, key phrases also accepted the `loggingOptOut` parameter. Here it only accepts modelVersion through PrebuiltTaskParameters. Should this be added?

Reply:
Good catch, I've just added `TaskParameters` to this object as well.

Reply:
Just want to point out that I think this change might have gotten missed

Reply:
Oops, thanks!  I have updated this one as well.  :)